lib/kino/explorer.ex

```diff
@@ -124,6 +124,7 @@ defmodule Kino.Explorer do
 
   defp most_frequent(data) do
     data
+    |> Series.mask(Series.is_not_nil(data))
```

Comment:
```suggestion
    |> Series.mask(Series.is_not_nil(data))
```

Another option is do `DataFrame.head(2)` and select the first column that is not nil. It may be slightly more efficient. Thoughts?

Reply:
Sorry, I am not sure I follow. How would this help in the case where we are trying to find the top occurring value of the series?

Reply:
`Series.frequencies` returns all values ordered by frequency. Our value will either be the first entry (if not nil) or the second entry (if the first is nil).

Reply:
ahh I get it, I guess that could be more performant, will change